index.d.ts

```diff
@@ -1,5 +1,6 @@
 declare module "prism-react-renderer" {
   import * as React from "react";
+  import type PrismJS from 'prismjs';
```

Comment:
Is there additional work needed to ensure this type definition gets included in the npm package?

Reply:
no additional work needed -- `index.d.ts` is already included in the tar